run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 ############################
 # Run Command:
-./mfnwt meras_transient_rjh_19Feb20.nam
+wine MODFLOW-NWT_64.exe CP_modflow.nam
```

Comment:
see comment above about maybe making template versions of these files to avoid tracking specifics

Reply:
yeah again i thought this was in .gitignore, don't know why it keeps slipping into the commits.